CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Fixed
 
+- [GH-1021] Use the cluster domain of undering k8s. No need to configure it explicitly.
```

Comment:
Use the cluster domain provided by kubelet instead of hardcoded `.cluster.local`